hack/e2e.go

```diff
@@ -18,6 +18,9 @@ limitations under the License.
 package main
 
 import (
+	"github.com/renstrom/dedent"
```

Comment:
nit, stdlib import block should be followed by project import (k8s.io/kubernetes) block which should be followed by vendor import block, each block seperated by one empty line.